src/css/json.css

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Why not just create that as global css via styled component rather than add all of this css loaders?